adsorbml/scripts/dense_eval.py

```diff
@@ -199,12 +199,7 @@ def filter_ml_data(ml_data, dft_data):
     new_ml_data = defaultdict(dict)
     for system in ml_data:
         for config in ml_data[system]:
-            if "rand" in config:
-                newconfig = "rand" + config.split("randpl")[1]
-                new_ml_data[system][newconfig] = ml_data[system][config]
-            else:
-                newconfig = "heur" + config.split("adslab")[1]
-                new_ml_data[system][newconfig] = ml_data[system][config]
+            new_ml_data[system][config] = ml_data[system][config]
```

Comment:
Now that we're not changing config names, I think L199-204 can be removed? `new_ml_data` should be the same as `ml_data`

Reply:
good call